build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -1690,4 +1690,15 @@ message RequestMetadata {
   // An identifier to tie multiple tool invocations together. For example,
   // runs of foo_test, bar_test and baz_test on a post-submit of a given patch.
   string correlated_invocations_id = 4;
+
+  // A one-word description of the kind of action, for example, CppCompile or GoLink.
+  // There is no standard agreed set of values for this, and they are expected to vary between different client tools.
+  string action_mnemonic = 5;
+
+  // A namespace within which the build was performed, for instance a repository/workspace name.
+  string invocation_namespace_id = 6;
+
+  // An identifier for the target, unique within the invocation_namespace_id, which produced this action.
+  // No guarantees are made around how many actions may relate to a single target.
+  string target = 7;
```

Comment:
The Build Event Stream identifies individual nodes in the build graph by a Bazel target and a Configuration ID:
https://github.com/bazelbuild/bazel/blob/dd8afa0fc79a7ff0fc0633eed6d592e0efd3bbfc/src/main/java/com/google/devtools/build/lib/buildeventstream/proto/build_event_stream.proto#L135-L138

The reason for this is that a single target can be compiled multiple times (e.g., through aspects, transitions, etc.). 

Should we also add this information to the metadata, so that we can correlate REv2 RPCs with build events?

Reply:
Sounds good - added